src/node_contextify.cc

```diff
@@ -935,6 +952,45 @@ class ContextifyScript : public BaseObject {
     return value->ToInteger(env->context());
   }
 
+  static MaybeLocal<Context> GetContext(Environment* env,
+                                        Local<Value> options) {
+    if (!options->IsObject())
+      return MaybeLocal<Context>();
+
+    // Symbol handle may be empty when initializing.
+    if (env->vm_parsing_context_symbol().IsEmpty())
+      return MaybeLocal<Context>();
+
+    MaybeLocal<Value> maybe_value =
+        options.As<Object>()->Get(env->context(),
+                                  env->vm_parsing_context_symbol());
+    if (maybe_value.IsEmpty())
+      return MaybeLocal<Context>();
+
+    Local<Value> value = maybe_value.ToLocalChecked();
+    if (!value->IsObject()) {
+      if (!value->IsNullOrUndefined()) {
+        env->ThrowTypeError(
+            "contextifiedSandbox argument must be an object.");
+      }
+      return MaybeLocal<Context>();
+    }
+
+    ContextifyContext* sandbox =
+        ContextifyContext::ContextFromContextifiedSandbox(
+            env, value.As<Object>());
+    if (!sandbox) {
+      env->ThrowTypeError(
+          "sandbox argument must have been converted to a context.");
+      return MaybeLocal<Context>();
+    }
+
+    Local<Context> context = sandbox->context();
+    if (context.IsEmpty())
+      return MaybeLocal<Context>();
```

Comment:
Not sure, but the same check is in a couple of other places in the file so I wouldn't touch them in this PR.